build.psm1

```diff
@@ -718,6 +727,36 @@ Fix steps:
     }
 }
 
+function Switch-PSNugetConfig {
+    param(
+        [ValidateSet('Public', 'Private')]
+        [string] $Source = 'Public'
+    )
+
+    if ( $Source -eq 'Public') {
+        $dotnetPerf = [NugetPackageSource] @{Url = 'https://pkgs.dev.azure.com/dnceng/public/_packaging/benchmark-dotnet-prerelease/nuget/v3/index.json'; Name = 'dotnet' }
```

Comment:
This one can be remove now. The package versions of BenchmarkDotNet are available in nuget.org now.